app/WikiDomain.php

```diff
@@ -39,4 +40,14 @@ public function wiki(): \Illuminate\Database\Eloquent\Relations\HasOne
     {
         return $this->hasOne(Wiki::class);
     }
+
+    static public function encode($domain)
```

Comment:
No objection to sticking this in here but as you said it's interesting that we don't really seem to use this model much. Maybe we ought to think about retiring it

Reply:
Writing a test rn - also I think maybe its own helper class would make sense

Reply:
Sure, as we discussed; using the help makes sense if you feel we really add some clarity with the tests